config/config.json

```diff
@@ -168,6 +168,7 @@
                 }
             }
         },
+        "maximumAssertionSizeInKb": 0,
```

Comment:
Do we want this to be zero, 2.5MB or somewhere inbetween?

Reply:
I am not sure that we want default value to be disabled this check, to me it makes more sense to put it at the highest allowed by the system (2.5)